pom.xml

```diff
@@ -540,6 +541,17 @@
         <artifactId>auto-value-annotations</artifactId>
         <version>${auto-value.version}</version>
       </dependency>
+
+    <!--
+    java-shared-config comes first in the release train. Defining the graal-sdk
+    version here will allow us manage the dependency from a single location
+    instead of needing to define it multiple repos (e.g gax, java-spanner)
+    -->
```

Comment:
Not sure this is necessary as it applies to everything in this pom - not just this new entry.